### PR TITLE
Option to get the defaultFS for ABFS from core-site else use the conf…

### PR DESCRIPTION
**Pull request by quadoss**
…ig from Hue

Jira: CDPD-39592

## What changes were proposed in this pull request?
Include a config to provide the user an option to see if they want to get the information from core-site or what they have set in the Hue config.

### CODE REVIEW

File: desktop/conf.dist/hue.ini [hunk 1]
```diff
@@ -1916,6 +1916,7 @@ submit_to=True
 [[abfs_clusters]]
 # Default ABFS cluster
 [[[default]]]
+## enable_defaultfs_from_coresite=true
```
Comment (Harshg999) [inline desktop/conf.dist/hue.ini:1919]: Maybe better to not add this config flag in the .ini files to let the config clean?

File: desktop/conf/pseudo-distributed.ini.tmpl [hunk 2]
```diff
@@ -1899,6 +1899,7 @@
   [[abfs_clusters]]
     # Default ABFS cluster
     [[[default]]]
+      ## enable_defaultfs_from_coresite=true
```
Comment (Harshg999) [inline desktop/conf/pseudo-distributed.ini.tmpl:1902]: Same, lets not add this flag here to let config be clean.

**Review by Harshg999 — APPROVED** [review]
Looks good! Few suggestions

File: desktop/conf.dist/hue.ini (hunk 1 above)
Comment (quadoss) [reply desktop/conf.dist/hue.ini:1919]: If a user wants to set the flag and if the flag does not exist in the ini file then the user will not know that there is an option available to set.

File: desktop/conf/pseudo-distributed.ini.tmpl (hunk 2 above)
Comment (quadoss) [reply desktop/conf/pseudo-distributed.ini.tmpl:1902]: If a user wants to set the flag and if the flag does not exist in the ini file then the user will not know that there is an option available to set.

**Pull request merged**